src/libdrakvuf/vmi.c

```diff
@@ -1103,10 +1129,14 @@ bool inject_trap_pa(drakvuf_t drakvuf,
     container->breakpoint.guard.memaccess.access = VMI_MEMACCESS_RW;
     container->breakpoint.guard.memaccess.type = PRE;
     container->breakpoint.guard.memaccess.gfn = current_gfn;
+    container->breakpoint.guard.ttl = UNLIMITED_TTL;
+    container->breakpoint.guard.ah_cb = NULL;
     container->breakpoint.guard2.type = MEMACCESS;
     container->breakpoint.guard2.memaccess.access = VMI_MEMACCESS_RWX;
     container->breakpoint.guard2.memaccess.type = PRE;
     container->breakpoint.guard2.memaccess.gfn = remapped_gfn->r;
+    container->breakpoint.guard2.ttl = UNLIMITED_TTL;
+    container->breakpoint.guard2.ah_cb = NULL;
```

Comment:
No `NULL` initialization should be necessary here, the `container` struct is already zero initialized.